### PR TITLE
Add automation for new PRs and issues

### PR DESCRIPTION
**Pull request by ripcurlx**
This should be a first small step to automate our PR label handling

### CODE REVIEW

**Comment by ripcurlx** [comment]
I recognized a problem in my local repo using this config so I'll need to do some more testing and might extend this PR to also handle the automatic `waiting for author` labeling.

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request, issues]
+
+jobs:
+    greeting:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/first-interaction@v1
+                with:
+                    repo-token: ${{ secrets.GITHUB_TOKEN }}
+                    issue-message: '**Thanks for opening your first issue here!**<br/><br/>Be sure to follow the issue template! Your issue will be reviewed by a maintainer and labeled accordingly for further actions.'
```
Comment (ripcurlx) [inline .github/workflows/greetings.yml]: @m52go Could you go over this messages and suggest improvements if required? Thanks!

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+                    pr-message: '**Thanks for opening this pull request!**<br/><br/>Please check out our [contributor checklist](https://docs.bisq.network/contributor-checklist.html) and have a look if issues that are found by *Travis* or *Codacy* running basic checks on your PR. Also make sure your commits are signed and you applied the Bisq code style and formatting.<br/><br/>A maintainer will add an `is:priority` label to your PR if it is up for compensation. Please checkout our [blog post](https://bisq.network/blog/TBD/) for more details on this.'
```
Comment (ripcurlx) [inline .github/workflows/greetings.yml]: @m52go Could you please also proof read this message? Are we planning to have a blog post on the new priorities as well?

Comment (m52go) [reply .github/workflows/greetings.yml]: Yes, although I'm not sure who/how that will be achieved...need to discuss with @cbeams.

File: .github/workflows/greetings.yml (hunk 1 above)
Comment (m52go) [inline .github/workflows/greetings.yml]:
```suggestion
                    issue-message: '**Thanks for opening your first issue here!**<br/><br/>Be sure to follow the issue template. Your issue will be reviewed by a maintainer and labeled for further action.'
```

Was a little bit too excited before ;)

File: .github/workflows/greetings.yml (hunk 2 above)
Comment (m52go) [inline .github/workflows/greetings.yml]:
```suggestion
                    pr-message: '**Thanks for opening this pull request!**<br/><br/>Please check out our [contributor checklist](https://docs.bisq.network/contributor-checklist.html) and check if *Travis* or *Codacy* found any issues with your PR. Also make sure your commits are signed, and that you applied Bisq code style and formatting.<br/><br/>A maintainer will add an `is:priority` label to your PR if it is up for compensation. Please see our [blog post](https://bisq.network/blog/TBD/) for more details.'
```

Basic edits.

Comment (m52go) [reply .github/workflows/greetings.yml]: Also, maybe link "Bisq code style and formatting" here:
https://github.com/bisq-network/style/issues

**Review by m52go — COMMENTED** [review]
Please see comments and suggestions for edits.

File: .github/workflows/greetings.yml (hunk 2 above)
Comment (ripcurlx) [reply .github/workflows/greetings.yml]: Thanks! Just force pushed the changes.

Comment (ripcurlx) [reply .github/workflows/greetings.yml]: Thanks! Just force pushed the changes.

File: .github/workflows/greetings.yml (hunk 1 above)
Comment (ripcurlx) [reply .github/workflows/greetings.yml]: Thanks! Just force pushed the changes.

**Comment by ripcurlx** [comment]
@m52go I'll leave this PR open until we've made the announcement and a decision on how we want to communicate this information best.

**Review by sqrrm — APPROVED** [review]
utACK

**Comment by m52go** [comment]
For the record, I corresponded with cbeams about this yesterday and we'll be publishing a blog post soon...will update this thread when it's merged.

**Pull request merged**